### ISSUE TITLE
masterInterval=false isn't being respected

### ISSUE THREAD

**Issue by alexgorbatchev** [grantcarthew/node-rethinkdb-job-queue#15]
```js
new Queue(connection, { 
  name: 'jobQueue',
  concurency: 10,
  masterInterval: false 
})
```

Produces the following log message

```
  [2016-10-03 00:52:24.304][db-review] enable +0ms 310000
```

However,

```js
new Queue(connection, { 
  name: 'jobQueue',
  concurency: 10,
  masterInterval: 10000 
})
```

Produces the following log message

```
  [2016-10-03 00:52:24.373][db-review] enable +0ms 10000
```

This is due to

```js
this._masterInterval = options.masterInterval ||
      enums.options.masterInterval
```

If `options.masterInterval` is `false`, it will default to `enums.options.masterInterval`. That should be something like

```js
  this._masterInterval = typeof options.masterInterval !== 'undefined'
    ? options.masterInterval
    : enums.options.masterInterval
    ;
```

**Comment by grantcarthew** [comment]
Good find on this one @alexgorbatchev. One of those changes I though I made but didn't.

Fixed in v0.4.1

Thanks.

**Comment by alexgorbatchev** [comment]
👍
